src/app/components/nested-data-picker/data/nested-value.interface.ts

```diff
@@ -9,5 +11,5 @@ export interface NestedValueInterface
     isVisible:boolean;
     name:string;
     tooltip:string;
-    tooltipPlacement:string;
+    tooltipPlacement:TerraPlacementEnum;
```

Comment:
does it work without changing typedef in `TerraNodeInterface`?

Reply:
No, my wrong